source/documentation/support/troubleshooting.md

```diff
@@ -8,30 +10,27 @@
 
 ### Common issues
 
-Please find information below about common problems when using registers. If your issue is not included or you are struggling to use registers in your product or service, please contact the [GDS registers team](https://registers.cloudapps.digital/support.html).
-
 **Registers still in beta**
 
-Despite the beta label, registers in beta are reliable to use in products and services.
+Despite the beta label, registers in beta are reliable and ready for use in products and services.
 
 **If you can't find a specific record**
 
-If you can't find a specific record, check you have the correct field value. You must use the exact field value to get a match from the register. For example, in the country register the field value must be in capital letters.
+If you can't find a specific record, check you have the correct field value. You must use the exact field value to get a match from the register. For example, in the `local-authority-eng` register, you must capitalise the field value: 
 
-Incorrect request: https://country.register.gov.uk/record/gb
+Incorrect request: https://local-authority-eng.register.gov.uk/record/bir
 
-Correct request: https://country.register.gov.uk/record/GB
+Correct request: https://local-authority-eng.register.gov.uk/record/BIR
 
-**End-date field**
+**Adding additional data**
 
-The `end-date` field is used to note when an entry ceases to be valid. For example, in the country register, the end-date field contains the date when a country was dissolved or changed its name. For example, the `end-date` field value for East Germany is `1990-10-02`.
+If you want to combine register data with data from other sources, you may find it useful to keep the data from each source in separate files and combine them at a later stage. For example, you could keep separate files for data about countries and for data about territories.
```

Comment:
I agree with you @arnau 

I wasn't convinced this belonged in the API docs, so I'm glad you brought it up. I would propose to merge this as-is since it's effectively only a minor tidy-up of some existing language, but to put this in for research.

Reply:
Yeah I agree that this doesn't make much sense. If we want to talk about combining registers with other data it's perhaps guidance that belongs somewhere else. Happy for this to be merged given that it was already there and add this to your list of things to address/research @bahmady